d3rlpy/base.py

```diff
@@ -340,6 +349,12 @@ def fit(self,
                     if val is not None:
                         logger.add_metric(name, val)
 
+                        # save loss to loss history dict
+                        if name not in self.loss_history_:
+                            self.loss_history_[name] = list()
```

Comment:
Probably we can omit this when you use `defaultdict`.

Reply:
exactly ! that's the intention to use defaultdict(list) to forget about that checks !

Reply:
Good to know. Thanks!